zone1/cli.c

```diff
@@ -478,9 +478,17 @@ void cliTask( void *pvParameters){
 			print_stats();
 		} else if (tk1 != NULL && strcmp(tk1, "restart")==0){
 			restart();
+		} else if (tk1 != NULL && strcmp(tk1, "exec")==0){
```

Comment:
This block seems to cause an exception when running stats:
```
Z1 > stats
11Load access fault : 0x00000005 0x8002b254 0x204136ea 
Illegal instruction : 0x00000002 0x0000fef4 0x204136ee 

Press any key to restart
```

The Illegal instruction comes from mepc being increment incorrectly in the handler.